### ISSUE TITLE
Raising exception leads to incorrect C code

### ISSUE THREAD

**Issue by plajjan** [actonlang/acton#111]
### Code

```Acton
def foo():
    raise ValueError("hejsan")

actor main(env):
    await async env.exit(0)
```

### Meta
`actonc --version`:
```
NOTE: acton 0.4.2.20210820.11.49.39 compiled by ghc 8.10 on linux x86_64
NOTE: cc: cc (Debian 10.2.1-6) 10.2.1 20210110
```

### Error output

```
kll@Boxy:~/terastream/acton/test/regression$ make 111-raise-exception 
Makefile:49: warning: overriding recipe for target '13-print-actor-method'
Makefile:34: warning: ignoring old recipe for target '13-print-actor-method'
../../dist/actonc 111-raise-exception.act --root main
/home/kll/terastream/acton/dist/modules/test/regression/111-raise-exception.c: In function ‘test$regression$_$111_45_raise_45_exception$$foo’:
/home/kll/terastream/acton/dist/modules/test/regression/111-raise-exception.c:3:5: warning: implicit declaration of function ‘$RAISE’; did you mean ‘RAISE’? [-Wimplicit-function-declaration]
    3 |     $RAISE((($Exception)$ValueError$new(to$str("hejsan"))));
      |     ^~~~~~
      |     RAISE
/usr/bin/ld: /home/kll/terastream/acton/dist/lib/libActonProject.a(test$regression$_$111_45_raise_45_exception.o): in function `test$regression$_$111_45_raise_45_exception$$foo':
/home/kll/terastream/acton/dist/modules/test/regression/111-raise-exception.c:3: undefined reference to `$RAISE'
collect2: error: ld returned 1 exit status
ERROR: internal compiler error: compilation of generated C code of the root actor failed
NOTE: this is likely a bug in actonc, please report this at:
NOTE: https://github.com/actonlang/acton/issues/new?template=ice.md
NOTE: acton 0.4.2.20210820.11.49.39 compiled by ghc 8.10 on linux x86_64
NOTE: cc: cc (Debian 10.2.1-6) 10.2.1 20210110
make: *** [Makefile:34: 111-raise-exception] Error 1
kll@Boxy:~/terastream/acton/test/regression$
```

I have tried writing my own Acton module in C and it appears that if I merely change `$RAISE` (which was generated here) to `RAISE`, it works - the Acton program runs and then exits with an error code + outputs the exception message to the terminal. So I guess this is a problem in CodeGen where we output a `$` too much.

**Comment by nordlander** [comment]
This one was also on my todo-list. The cause is a simple naming confusion, but it's the builtin/rts C files that must change.

On its way.

**Comment by nordlander** [comment]
Resolved by #130
